infra/faucet/src/scripts/airDrop.js

```diff
@@ -0,0 +1,215 @@
+// Tool to distribute OGN tokens to a list of ETH addresses stored in a text file.
+// Command line example:
+//   node airDrop.js --recipientsFilename=addresses.txt --networkId=999 --campaignId=1
+
+'use strict'
+
+const BigNumber = require('bignumber.js')
+const fs = require('fs')
+const Logger = require('logplease')
+const Web3 = require('web3')
+
+const Token = require('@origin/token/src/token')
+const { createProviders } = require('@origin/token/src/config')
+
+const enums = require('../enums')
+const db = require('../models')
+
+Logger.setLogLevel(process.env.LOG_LEVEL || 'INFO')
+const logger = Logger.create('airdrop')
+
+/**
+ * Parse command line arguments into a dict.
+ * @returns {Object} - Parsed arguments.
+ */
+function parseArgv() {
+  const args = {}
+  for (const arg of process.argv) {
+    const elems = arg.split('=')
+    const key = elems[0]
+    const val = elems.length > 1 ? elems[1] : true
+    args[key] = val
+  }
+  return args
+}
+
+/**
+ * Reads a file having an eth address on each line
+ * @param filename
+ * @returns {string[]}
+ */
+function readRecipients(filename) {
+  const data = fs.readFileSync(filename).toString()
+  const addresses = data
+    .split('\n')
+    .map(l => l.trim().toLowerCase())
+    .filter(l => Boolean(l))
+  const seen = {}
+  for (const address of addresses) {
+    if (!Web3.utils.isAddress(address)) {
+      throw new Error(`Invalid address: ${address}`)
+    }
+    if (seen[address]) {
```

Comment:
Maybe standardise to lower case for the address? Could it be that the same address exists multiple times with different casing?

Reply:
Ahh nevermind I see you have above